assets/src/edit-story/integrations/wordpress/metaBoxes/metaBoxesArea.js

```diff
@@ -43,11 +43,6 @@ const Wrapper = styled.div`
     box-sizing: content-box;
   }
 
-  textarea,
-  input {
-    box-sizing: border-box;
-  }
-
```

Comment:
Are we sure these aren't needed anymore? Are they still children of a relevant (fixed) node or are they somewhere outside the tree? @swissspidy?

Reply:
Would need to double check by toggling the meta box area and see if everything is still in order.

Reply:
Added the `All in One SEO Pack` plugin. There's a lot of stuff there but it is getting the box sizing styles: 
<img width="1294" alt="Screen Shot 2021-03-17 at 9 44 22 AM" src="https://user-images.githubusercontent.com/22185279/111495912-6e4c1c00-8705-11eb-9c2f-4f600069f4fb.png">